examples/airblog/airblog.py

```diff
@@ -21,9 +9,9 @@
 from operator import itemgetter
 from pathlib import Path
 
-import mistletoe
+import mistletoe  # type: ignore[import-untyped]
```

Comment:
Remove the type ignore and add to line 451 of Air's root pyproject.toml this:

```toml
"examples/airblog/**"
```

Reply:
done!